ansible/roles/create-vm/tasks/main.yml

```diff
@@ -2,12 +2,12 @@
 
 - name: Check family excluded
   set_fact:
-    excluded: "{{ [vm_family, vm_collection_method] in excluded_pairs }}"
+    excluded: "{{ [vm_family, vm_collection_method] in excluded_pairs or (test_platform != 'rhel-ppc64le' and vm_collection_method == 'ebpf') }}"
   when: vm_image is not defined
 
 - name: Check image excluded
   set_fact:
-    excluded: "{{ [vm_image, vm_collection_method] in excluded_pairs }}"
+    excluded: "{{ [vm_image, vm_collection_method] in excluded_pairs or (test_platform != 'rhel-ppc64le' and vm_collection_method == 'ebpf') }}"
   when: vm_image is defined
 
 - name: Create IBM Z VM
```

Comment:
We don't need to do this in this PR, but it looks like all tasks starting with this one could be added to a block and have a global `when: excluded == false` instead of checking on every single one, (except for the very last task that run unconditionally).